README.md

```diff
@@ -837,7 +837,8 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
 
 **[`^        back to top        ^`](#)**
 
-  * [Chevereto 2.X](https://github.com/Chevereto/Chevereto-2.X) - Open source one-click image hosting script. ([Source Code](https://github.com/Chevereto/Chevereto-2.X)) `MIT` `PHP`
+  * [Chevereto Free](https://github.com/Chevereto/Chevereto-Free) - A powerful and fast image hosting script that allows you to create your very own full featured image hosting website in just minutes. ([Source Code](https://github.com/Chevereto/Chevereto-Free)) `AGPLv3` `PHP`
```

Comment:
I'm not duplicating links. Two different repos, two different systems under different licensing (agpl / MIT). 

Anyway, feel free to remove Chevereto-2.X since that edition is not under active development. 

Thanks.